code_search/src/code_search/dataflow/cli/arguments.py

```diff
@@ -29,6 +29,8 @@ def add_parser_arguments(parser):
                       help='BigQuery dataset for output results')
   additional_args_parser.add_argument('--pre_transformed', action='store_true',
                       help='Use a pre-transformed BigQuery dataset')
+  additional_args_parser.add_argument('--wait_till_finish', action='store_true',
```

Comment:
Nit can you name this "wait_until_finish" to match Dataflow nomenclature.

Reply:
done.